test/fixtures/v5/end_of_road/sharp_left_destination.json

```diff
@@ -10,7 +10,7 @@
     },
     "instructions": {
         "ar": "انعطف يسار حادا نحو Destination 1",
-        "da": "Drej til skarpt venstresving mof Destination 1",
+        "da": "Drej til skarpt venstresving mod Destination 1",
```

Comment:
Should be "Drej skarpt til venstre mod Destination 1"

Reply:
Yes, required `modifier` 2-words value decomposing here and below could be done with post-processing only but maybe it's possible also to change its value right in Transifex to
```
"sharp left": "skarpt venstre"
```
?
Or there are other cases where such modifiers should be inserted in previous full form?

Reply:
But it should be `skarpt til venstre`. Or would you add `til` in post-processing?

Reply:
You could move `til` into most modifiers, like `"left": "til venstre"` and `"sharp left": "skarpt til venstre""`, and then in the `turn` instructions you could just say `drej {modifier}`.
This would fix some of the issues, but not all.

Reply:
If this `til` moving right into all `modifier` values will fix most problems, I'd recommend to do this - then it will be much easier to solve rest issues 😉

Reply:
@yuryleb I can do that, but it's kind of weird that "left" will be translated as "to the left" in Danish. But if you don't mind, then I'll do it